ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -1,146 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- *
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
- * General Public License as published by the Free Software Foundation, either version 3 of the
- * License, or any later version.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
- * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
- * is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- *
- **/
-/*global require*/
-import React from 'react'
-import leftPad from '../../react-component/utils/left-pad'
-
-const Marionette = require('marionette')
-const CustomElements = require('../../js/CustomElements.js')
-const user = require('../singletons/user-instance.js')
-const properties = require('properties')
-const metacardDefinitions = require('component/singletons/metacard-definitions')
-const Common = require('js/Common')
-const hbHelper = require('js/HandlebarsHelpers')
-
-const mtgeo = require('mt-geo')
-const usngs = require('usng.js')
-
-const converter = new usngs.Converter()
-const usngPrecision = 6
-
-function getCoordinateFormat() {
-  return user
-    .get('user')
-    .get('preferences')
-    .get('coordinateFormat')
-}
-
-function formatAttribute(attributeName, attributeValue) {
-  if (metacardDefinitions.metacardTypes[attributeName].type === 'DATE') {
-    attributeValue = Common.getHumanReadableDateTime(attributeValue)
-  }
-  return attributeName.toUpperCase() + ': ' + attributeValue
-}
-
-module.exports = Marionette.LayoutView.extend({
-  template() {
-    return (
-      <React.Fragment>
-        {this.getAttributes()}
-        <div className="info-feature">{this.target}</div>
-        <div className="info-coordinates">{this.getDisplayComponent()}</div>
-      </React.Fragment>
-    )
-  },
-  tagName: CustomElements.register('map-info'),
-  modelEvents: {},
```

Comment:
are the empty curly braces on lines 58, 62, and 63 safe to ignore for the migration?